src/Model/ModelManagerInterface.php

```diff
@@ -24,7 +24,8 @@
  *
  * A model manager is a bridge between the model classes and the admin functionality.
  *
- * @method bool supportsQuery(object $query)
+ * @method bool   supportsQuery(object $query)
+ * @method object reverseTransform(object $object, array $array = [])
```

Comment:
Is this method intended to always return the same instance received in argument 1?

Reply:
I would say yes, why ?

Reply:
Not a blocker at all, but in that case, I think there is no need to return the same instance. The method can return `void` and make the required mutations on the received object, since them will be kept event out of the scope of this method.